pkgs/by-name/un/unimatrix/package.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, python3
+, fetchFromGitHub
+, git
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "unimatrix";
+  version = "unstable-2023-04-25";
+
+  src = fetchFromGitHub {
+    owner = "will8211";
+    repo = pname;
+    rev = "65793c237553bf657af2f2248d2a2dc84169f5c4";
+    hash = "sha256-MtHsoZvUpYPQ5O9CJC7TwcOQOT4ATfyNpoozWYSlI7Q=";
+    # Needed for setuptools-scm to detect version
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [
+    python3.pkgs.setuptools-scm
+    python3.pkgs.pip
+    git
+  ];
+
+  meta = with lib; {
+    description = "Python script to simulate the display from \"The Matrix\" in terminal.";
```

Let's avoid the escaped quotes 

```suggestion
    description = ''Python script to simulate the display from "The Matrix" in terminal'';
```

---

Done